src/types/json.h

```diff
@@ -144,6 +146,50 @@ struct JsonValue {
     return Status::OK();
   }
 
+  Status StrAppend(std::string_view path, const std::string &append_value, std::vector<uint64_t> &append_cnt) {
+    try {
+      std::string append_trimmed;
```

Comment:
I think `append_str` may be a better name.

Reply:
but, other functions in `json.h` use Camel-Case， like `ArrAppend`

Reply:
I mean the variable name.